ksql-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlan.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.planned;
+
+import io.confluent.ksql.engine.KsqlPlan;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public final class TestCasePlan {
+  private final TestCasePlanNode node;
```

Comment:
Is this `TestCasePlan`/`TestCasePlanNode` naming practice standard? I was surprised by this since given the names I would've expected that a `TestCasePlanNode` contains a `TestCasePlan` and a topology, rather than a `TestCasePlan` containing a `TestCasePlanNode` and a topology.

Reply:
in this module `*Node` usually refers to a json-serialized pojo. A TestCasePlan is serialized as a json document and a topology file. TestCasePlanNode is the json part.